dom.go

```diff
@@ -2,18 +2,44 @@ package vecty
 
 import (
 	"fmt"
+	"reflect"
 
 	"github.com/gopherjs/gopherjs/js"
 )
 
-// Core implements the Context method of the Component interface, and is the
-// core/central struct which all Component implementations should embed.
+var (
+	errMissingParent = fmt.Errorf("missing parent node")
+	errEmptyElement  = fmt.Errorf("empty element or node")
```

Comment:
Also, minor, since these are static errors and you're not using any %v verbs, you could use the simpler `errors.New` instead of `fmt.Errorf`.